lib/logger.js

```diff
@@ -222,22 +222,21 @@ Logger.prototype._bufferLog = function(message) {
 
     if (!this._flusher) {
         debug('No scheduled flush. Scheduling for %d ms from now.', configs.FLUSH_INTERVAL);
-        this._flusher = setTimeout(bind(this._flush, this, (err) => {
-            if (err) {
-                debug('Received an error while flushing...' + err);
+        this._flusher = setTimeout(() => {
+            if (this._buf.length > 0) {
+                this._flush((e) => {debug(e);});
             }
-        }), this._flushInterval);
+        }, this._flushInterval);
     }
 };
 
 Logger.prototype._flush = function(cb) {
     if (this._buf.length === 0) {
-        debug('Nothing to flush');
-        return cb && cb();
+        // Warn("Nothing to flush")
```

Comment:
we don't really need this here as a comment ya?

Reply:
@vilyapilya, remove the line instead of commenting out